features/integration/algod.feature

```diff
@@ -20,8 +20,8 @@ Feature: Algod
     Given a kmd client
     And wallet information
     Then I get transactions by address and round
-    And I get transactions by address only
-    And I get transactions by address and date
+    # And I get transactions by address only
+    # And I get transactions by address and date
```

Comment:
requiring indexer v1

Reply:
@tzaffi Is it your preference to comment out instead of delete the indexer v1 features?

I'm willing to be convinced, but my default is to suggest deleting it.  My thinking is future maintainers will have difficulty understanding _why_ these lines are commented out.